tools/README.md

```diff
@@ -1,24 +1,21 @@
-# Tools
+# Google CloudEvent Repo Tools
 
 The tools in this directory are not required in order to *use* the
 rest of the content of this repository. They are present to help
 create or maintain the content.
 
-While they are *currently* written in C#, requiring .NET Core 3.1 to
-build and run, this is purely incidental. Pull requests to rewrite
-the tools in languages more commonly used at Google (e.g. Java, Go
-or Python) are welcome.
+## [readme-list](./readme-list/readme.md)
 
-## registry
+This tool edits the top-level [README](../README.md) event list.
```

Comment:
Will you send a later PR that removes the C# code then? (To confirm: I don't see any reason to keep it.)

Reply:
Ah - I see this change was reverted in the second commit. I don't mind whether the C# code stays (if it's simpler than rewriting it) or goes (if the README editing drops out naturally).

Reply:
Yeah, I started reworking the registry based off of our JSON schemas, but then stopped as I didn't want to implement too much in one PR.

Will send a follow-up PR for the registry part.